src/js/themes/base.js

```diff
@@ -840,6 +840,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // extend: undefined,
       icons: {
         down: FormDown,
+        up: FormDown,
```

Comment:
```suggestion
        // up: undefined,
```
Let's keep it undefined unless users set it otherwise.
This change will impact the way the icon is being resolved on Menu.js.

Reply:
Yes, agree, it's logical to keep this property undefined unless the user set it manually. I will change Menu.js soon.
And I have 1 question. When I was running tests using 'npm test' command, some of the tests weren't passing, and I've updated snapshots ('npm test -- -u') and committed them. But after that, received failure on CircleCI tests. Can you advise how to solve this? Revert the previous commit and add commit without conflicted files?

Reply:
When there are unrelated snapshot changes to your PR it's mostly because you are using the wrong Node version, try to use at least version 12.
If that doesn't fix the snapshots, remove the irrelevant snapshots, and keep just the one for Menu. Hoping that will work.

Reply:
I installed 12.18.2 node, changed snapshots but they still can't pass CircleCI. And I can't remove changes in these snapshots and commit them, git doesn't allow to commit these changes in files. I tried to fix it manually from GitHub, but have forgotten to sign off my commits(. So I can just close this pr and open new with only changes, which are required. I think it's the simplest way to resolve this.

Reply:
No worries, I was able to fix it.